core/src/components/modal/modal.ionic.scss

```
@@ -0,0 +1,19 @@
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+@import "./modal";
```

Typically we have been splitting the common file (`modal.scss`) into two files `modal.native.scss` and `modal.common.scss` and then importing the common one instead:

```diff
-@import "./modal";
+@use "./modal.common";
```

However, since this PR is only adding shape, I don't think it makes sense to try to figure out what the values in `core/src/components/modal/modal.vars.scss` will be for the Ionic theme. We have a ticket (ROU-11153) for adding the default styling, so I think we can handle that there. OPTIONAL suggestion to keep track of this:

```suggestion
// TODO(ROU-11153): separate modal styles to common and native
@import "./modal";
```